### PR TITLE
Improvments for explorer

### PR DESCRIPTION
**Pull request by 4miners**

### CODE REVIEW

**Comment by karmacoma** [comment]
Thank you very much @4miners. I will review asap.

**Comment by Gr33nDrag0n69** [comment]
nice job mq

**Comment by simonmorgenthaler** [comment]
very nice mq

File: modules/transactions.js [hunk 1]
```diff
@@ -187,6 +187,33 @@ __private.getById = function (id, cb) {
...
+		if (!rows.length) {
+			return setImmediate(cb, 'Transaction not found: ' + id);
+		}
+
+		var votes = rows[0].votes.split(',');
+		var added = [];
+		var deleted = [];
+
+		for (var i = 0; i < votes.length; i++) {
+			if (votes[i].substring(0, 1) == "+") {
```
Comment (vidul-nikolaev-petrov) [inline modules/transactions.js]: Please consider string indexing ([here](https://jsperf.com/index-vs-slice-vs-substring) is a performance test).

Comment (4miners) [inline modules/transactions.js]: Difference in performance is negligible, so more like personal preference.

Comment (4miners) [inline modules/transactions.js]: I getting inconsistent results, but those tests are for browsers and code is for node.js, so they are not meaningful. I would be grateful if you can you test it under node.js.

Comment (vidul-nikolaev-petrov) [inline modules/transactions.js]: Sure: [gist](https://gist.github.com/vidul-nikolaev-petrov/a6f615373bbb0fd3fd985418926606fa) and [online](https://goo.gl/VTDzzO
) example. Tested on localhost on _v4.6.1_ and _v6.9.0_ -- similar results.

Comment (4miners) [inline modules/transactions.js]: Thank you, that seems fair enough.

**Pull request merged**